CHANGELOG.md

```diff
@@ -28,12 +33,40 @@ Deprecated features
   ```
   The old files still exist and re-export the contents of the new modules.
 
+* In `Relation.Binary.NonStrictToStrict`:
+  ```agda
+  irrefl         ↦ <-irrefl
+  trans          ↦ <-trans
+  antisym⟶asym ↦ <-asym
+  decidable      ↦ <-decidable
+  trichotomous   ↦ <-trichotomous
+
+  isPartialOrder⟶isStrictPartialOrder ↦ <-isStrictPartialOrder
+  isTotalOrder⟶isStrictTotalOrder     ↦ <-isStrictTotalOrder₁
+  isDecTotalOrder⟶isStrictTotalOrder  ↦ <-isStrictTotalOrder₂
+  ```
+
 Removed features
 ----------------
 
 Backwards compatible changes
 ----------------------------
 
+<<<<<<< HEAD
```

Comment:
Looks like there were problems with the merge

Reply:
Eek. Didn't realise there were two. Fixed by b5345d7